docs/getting-started/release_package.md

```diff
@@ -128,4 +141,11 @@
        OS="linux"
        ARCH="arm"
        curl -L "https://github.com/kubeedge/kubeedge/releases/download/${VERSION}/kubeedge-${VERSION}-${OS}-${ARCH}.tar.gz" --output kubeedge-${VERSION}-${OS}-${ARCH}.tar.gz && tar -xf kubeedge-${VERSION}-${OS}-${ARCH}.tar.gz  -C /etc
-   ``` 
\ No newline at end of file
+   ```
+   + Observing CRI runtime containers
```

Comment:
Observing CRI runtime containers -> Monitoring containers status

If the container runtime configured to manage containers is containerd , then the following commands can be used to inspect container status and list images.